libsolidity/interface/FileReader.h

```diff
@@ -90,11 +90,16 @@ class FileReader
 
 	/// Normalizes a filesystem path to make it include all components up to the filesystem root,
 	/// remove small, inconsequential differences that do not affect the meaning and make it look
-	/// the same on all platforms (if possible). Symlinks in the path are not resolved.
+	/// the same on all platforms (if possible).
 	/// The resulting path uses forward slashes as path separators, has no redundant separators,
 	/// has no redundant . or .. segments and has no root name if removing it does not change the meaning.
 	/// The path does not have to actually exist.
-	static boost::filesystem::path normalizeCLIPathForVFS(boost::filesystem::path const& _path);
+	/// @param _path Path to normalize.
+	/// @param _resolveSymlinks If false, any symlinks present in @a _path are preserved.
```

Comment:
Is that a special doxygen syntax for arrays or something?

I meant that if you set it to `false` and for example `/a/b/c` is a symlink pointing at `/d/e/f` then `/a/b/c/x/y/z.sol` in `_path` will be normalized into `/a/b/c/x/y/z.sol`. Otherwise it will be normalized into `/d/e/f/x/y/z.sol`.

Reply:
I am not sure, either. Whether `@a` is special doxygen syntax or not. I assumed it's just referencing the parameter and for that it would need the `@` and didn't understand why it's on the `a` :)

Reply:
[`@a`](https://www.doxygen.nl/manual/commands.html#cmda) is just emphasis without special semantic meaning.

> Displays the argument <word> in italics. Use this command to emphasize words. Use this command to refer to member arguments in the running text.

I'd rather be using backticks but I don't think they're supported in the docstring syntax :)